crates/oxc_minifier/src/ast_passes/peephole_substitute_alternate_syntax.rs

```diff
@@ -89,6 +91,11 @@ impl<'a> Traverse<'a> for PeepholeSubstituteAlternateSyntax {
         }
     }
 
+    fn exit_expression(&mut self, expr: &mut Expression<'a>, ctx: &mut TraverseCtx<'a>) {
+        self.compress_new_expression(expr, ctx);
+        self.compress_call_expression(expr, ctx);
```

Comment:
I recommend using the `match expr` pattern, and then pass `NewExpression<‘a>`, etc. type to the compressor function with the return `Expression<'a>`. Subsequently, replace these objects in the `Traverse` function. This approach allows us to align the code style of the replacement, enhancing the code’s sustainability.